charts/akeyless-k8s-secrets-injection/values.yaml

```diff
@@ -11,8 +11,8 @@ tlsCertsSecretName: vault-secrets-webhook-tls-certs
 
 image:
   repository: akeyless/k8s-webhook-server
+  agentImage: akeyless/k8s-secrets-sidecar
```

Comment:
it always on latest tag?

Reply:
the version is taken from Chart.AppVersion